### ISSUE TITLE
Only allow logging when wanted

### ISSUE THREAD

**Issue by jrcastillo** [pinojs/hapi-pino#9]
I just started using this library (is awesome BTW), but I have a question, is there anyway to restrict the library from logging at all the events explained at the `README.md` ?

I wish to set the log at certain moments and not take into consideration those events specified by the documentation. Also this brings up to mind, if I should really use this extension or use `pino` instead.
Also I wish to set different log files for each type logging level and also maintain a general log. (I'm new to logging but must do a hell of a good job)

What do you guys recommend ? (I'm using a hapi server ... just to make sure its clear)

**Comment by mcollina** [comment]
Can you make some examples of what you do not want to log? Just to understand better what your issue is.

**Comment by jrcastillo** [comment]
Here is the example of how I currently configured the `hapi-pino` extension:
`server.js`
```js
let fs = require('fs');
let writeStream = fs.createWriteStream('output.log');

var plugins = [
  {
    register : require('hapi-pino'),
    options  : {
      stream : writeStream
    }
  }
];

server.register(plugins, function(err){
    //some other configs
    server.app.logger.warn('Pino is registered');
    server.start(function() {
       //some code
    });
})
```
and without doing anything else, this turns out to be my `output.log`
```json
[2016-08-16T17:03:10.732Z] WARN (6223 on comp): Pino is registered
[2016-08-16T17:03:10.747Z] INFO (6223 on comp): server started
    created: 1471366990481
    started: 1471366990740
    host: "comp"
    port: 3000
    protocol: "http"
    id: "comp:6223:irxpwzv5"
    uri: "http://comp:3000"
    address: "0.0.0.0"
[2016-08-16T17:03:21.832Z] INFO (6223 on comp): request completed
    res: {
      "statusCode": 200,
      "header": "HTTP/1.1 200 OK\r\nvary: origin,accept-encoding\r\ncontent-type: application/json; charset=utf-8\r\ncache-control: no-cache\r\ncontent-encoding: gzip\r\nDate: Tue, 16 Aug 2016 17:03:21 GMT\r\nConnection: keep-alive\r\nTransfer-Encoding: chunked\r\n\r\n"
    }
    responseTime: 64
    req: {
      "id": "1471367001768:comp:6223:irxpwzv5:10000",
      "method": "GET",
      "url": "/categories",
      "headers": {
        "host": "localhost:3000",
        "connection": "keep-alive",
        "cache-control": "max-age=0",
        "upgrade-insecure-requests": "1",
        "user-agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.82 Safari/537.36",
        "accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8",
        "accept-encoding": "gzip, deflate, sdch",
        "accept-language": "es,en-US;q=0.8,en;q=0.6"
      },
      "remoteAddress": "127.0.0.1",
      "remotePort": 38732
    }
[2016-08-16T17:03:39.804Z] INFO (6223 on comp): request completed
    res: {
      "statusCode": 200,
      "header": "HTTP/1.1 200 OK\r\nvary: origin,accept-encoding\r\ncontent-type: application/json; charset=utf-8\r\ncache-control: no-cache\r\ncontent-encoding: gzip\r\nDate: Tue, 16 Aug 2016 17:03:39 GMT\r\nConnection: keep-alive\r\nTransfer-Encoding: chunked\r\n\r\n"
    }
    responseTime: 21
    req: {
      "id": "1471367019783:comp:6223:irxpwzv5:10001",
      "method": "GET",
      "url": "/users",
      "headers": {
        "host": "localhost:3000",
        "connection": "keep-alive",
        "upgrade-insecure-requests": "1",
        "user-agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.82 Safari/537.36",
        "accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8",
        "accept-encoding": "gzip, deflate, sdch",
        "accept-language": "es,en-US;q=0.8,en;q=0.6"
      },
      "remoteAddress": "127.0.0.1",
      "remotePort": 38732
    }
```
As you can see, the log file contains log requests, responses and contains many others but it would make the example really big so I shortened it out.

I don't wish to log this kind of operations, I would just want to log errors, and operations I only wish to keep track off, depending on the level and on seperate files. For example:

fatalError.log:
```
[2016-08-16T17:09:32.260Z] FATAL (6335 on comp): Fatal error {...}
```
request.log:
```
[2016-08-16T17:09:32.260Z] INFO (6335 on comp): request {...}
```
warnings.log:
```
[2016-08-16T17:09:32.260Z] WARN (6335 on comp): request {...}
```
Thanks for the fast reply

**Comment by mcollina** [comment]
A couple of things:

a. pino does not support saving different levels to different files. That logic slows things down. It should be fairly easy to write another little tool that does that so you can pipe them (if you do, send a PR to the pino README with a link).

b. I don't think it's currently possible to disable request logging, but I'm open to a pull request in that regard, basically we should be able to customize the level of https://github.com/mcollina/hapi-pino/blob/master/index.js#L74-L80.

**Comment by jrcastillo** [comment]
As soon as I can I'll have a look at the code and check to see if I can add the PR with the customization of levels to log.

**Comment by juanjoLenero** [comment]
I could look into this if it's still needed/wanted.

Maybe a plugin-level option: `logEvents`.

```js
{
    register: 'hapi-pino',
    options: {
       logEvents: ['server-start', 'server-stop', 'response', 'request-error'] // Default
    }
}
```
Omitting any of the events would disable that specific log callback in hapi-pino.

**Comment by mcollina** [comment]
@juanjoLenero 👍 , would you like to send a PR?

**Comment by mcollina** [comment]
Fixes by #17.
